imap_processing/hit/l0/decom_hit.py

```diff
@@ -0,0 +1,464 @@
+"""Decommutate HIT CCSDS science data."""
+
+from collections import namedtuple
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+
+from imap_processing import imap_module_directory
+from imap_processing.utils import packet_file_to_datasets
+
+HITPacking = namedtuple(
+    "HITPacking",
+    [
+        "bit_length",
+        "section_length",
+        "shape",
+    ],
+)
+
+COUNTS_DATA_STRUCTURE = {
+    # field: bit_length, section_length, shape
+    # ------------------------------------------
+    # science frame header
+    "hdr_unit_num": HITPacking(2, 2, (1,)),
+    "hdr_frame_version": HITPacking(6, 6, (1,)),
+    "hdr_status_bits": HITPacking(8, 8, (1,)),
+    "hdr_minute_cnt": HITPacking(8, 8, (1,)),
+    # ------------------------------------------
+    # spare
+    "spare": HITPacking(24, 24, (1,)),
+    # ------------------------------------------
+    # erates
+    "livetime": HITPacking(16, 16, (1,)),
+    "num_trig": HITPacking(16, 16, (1,)),
+    "num_reject": HITPacking(16, 16, (1,)),
+    "num_acc_w_pha": HITPacking(16, 16, (1,)),
+    "num_acc_no_pha": HITPacking(16, 16, (1,)),
+    "num_haz_trig": HITPacking(16, 16, (1,)),
+    "num_haz_reject": HITPacking(16, 16, (1,)),
+    "num_haz_acc_w_pha": HITPacking(16, 16, (1,)),
+    "num_haz_acc_no_pha": HITPacking(16, 16, (1,)),
+    # -------------------------------------------
```

Comment:
can you add comment here what this section is? Above abbr. comments were still helpful!